auth-web/src/components/auth/staff/DissolutionSchedule.vue

```diff
@@ -0,0 +1,296 @@
+                  </template>
+
+                  <v-list dense>
+                    <v-list-item @click="triggerEditOnOff()">
+                      <v-list-item-subtitle>
+                        <v-icon>mdi-pencil</v-icon>
+                        <span class="pl-2 edit-txt">Edit</span>
+                      </v-list-item-subtitle>
+                    </v-list-item>
+                  </v-list>
+                </v-menu>
+              </span>
+            </v-col>
+          </template>
+        </v-row>
+
+        <!-- Cancel/Save buttons -->
+        <template v-if="isEdit">
+          <v-divider class="mb-8 mt-8" />
+          <v-row no-gutters>
+            <v-col
+              cols="12"
+              class="text-right"
+            >
+              <v-btn
+                color="primary"
+                large
+                outlined
+                class="mr-3"
+                @click="triggerEditOnOff()"
+              >
+                Cancel
+              </v-btn>
+              <v-btn
+                color="primary"
+                large
+                @click="saveBtnClicked()"
+              >
+                Save
+              </v-btn>
+            </v-col>
+          </v-row>
+        </template>
+      </article>
+    </section>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Vue } from 'vue-property-decorator'
+import { mapActions } from 'pinia'
+import { useStaffStore } from '@/stores/staff'
+
+@Component({
+  methods: {
+    ...mapActions(useStaffStore,
+      ['getDissolutionBatchSize',
+        'isDissolutionBatchOnHold',
+        'updateDissolutionBatchSize',
+        'updateDissolutionBatchOnHold'])
+  }
+})
+export default class DissolutionSchedule extends Vue {
+  $refs: {
+    numberOfBusinessesRef: any
+  }
+
+  // Local properties
+  menu = false
+  numberOfBusinesses = 0
+  isEdit = false
+  isOnHold = false
+
+  readonly getDissolutionBatchSize!: () => number
+  readonly isDissolutionBatchOnHold!: () => boolean
+  readonly updateDissolutionBatchSize!: (x: number) => void
+  readonly updateDissolutionBatchOnHold!: (x: boolean) => void
+
+  /**
+   * Set local properties to values from the store.
+   * TODO: Update once BE work is done.
+   */
+  mounted (): void {
+    this.numberOfBusinesses = this.getDissolutionBatchSize()
+    this.isOnHold = this.isDissolutionBatchOnHold()
+  }
+
+  /**
+   * Update whether the dissolution batch is paused or running.
+   * Emit the status to the parent to know whether the "Paused" badge is going to be shown.
+   * TODO: Modify/Update this once the BE is done. */
+  actionBtnClicked (): void {
+    this.isOnHold = !this.isDissolutionBatchOnHold()
+    this.onHoldUpdate(this.isOnHold)
+    this.updateDissolutionBatchOnHold(!this.isDissolutionBatchOnHold())
+  }
+
+  /**
+   * Save button is clicked. Update the dissolution batch size.
+   * Only save if the inputted number is valid.
+   * TODO: Implement logic (job) once the BE is done.
```

Comment:
How are you keeping track of this TODO?

Reply:
This comment is in a lot of places. Do you have a follow-up ticket with todos for all places that need to be updated?

Reply:
The TODOs will all be worked on in this ticket: https://github.com/bcgov/entity/issues/20929

I'm going to grab it and work on it eventually. So, in my head, I have everything in place 😆